Scripts/ParseEmailFiles/ParseEmailFiles.yml

```diff
@@ -81,6 +81,6 @@ outputs:
 scripttarget: 0
 runonce: false
 runas: DBotWeakRole
-releaseNotes: Fix - support email files with type 'SMTP Mail'
-tests:
-  - "ParseEmailFiles-test"
```

Comment:
Why did you remove the test playbook?

Reply:
because the pycharm plugin always reorders my yml, I copy yml from Demisto each time. I forget to add it back. Will add